### DISCUSSION TITLE
Cannot open page in Vietnam

### DISCUSSION THREAD

**Discussion by websajang** [vercel/next.js#44925] [Help]
I have made an online shop webpage for vietnamese people but visiting here the country I see that the page doesnt load here, but it does with an vpn connecting to Europe. Anoyone has any idea about the reason?

**Comment by ciruz** [comment]
Where is it hosted, do you have a link?

**Comment by websajang** [comment]
it is hosted in Hong Kong server with Vercel and the domain was bought also
from vercel.
khuyenvanong.com
